packages/panels/resources/views/components/page/sub-navigation/sidebar.blade.php

```diff
@@ -8,6 +8,7 @@
 >
     @foreach ($navigation as $navigationGroup)
         <x-filament-panels::sidebar.group
+            :attributes="\Filament\Support\prepare_inherited_attributes($navigationGroup->getExtraAttributeBag())"
```

Comment:
Please move `:attributes` to be the last one in the list of attributes (like we do elsewhere).